src/components/StudentDashboard.jsx

```diff
@@ -101,13 +101,13 @@ class StudentDashboard extends Component {
   generateDepartmentForm() {
     let departments = [];
     this.state.departments.forEach(department => {
-      let id = Array.join(department.split(" "));
+      // let id = Array.join(department.split(" "));
```

Comment:
This line was giving a compilation error for me. Is the `id` field used anywhere?